packages/teams-js/src/public/meeting.ts

```diff
@@ -94,29 +94,68 @@ export namespace meeting {
     type?: T;
   }
 
+  /**
+   * @hidden
+   * Hide from docs
+   * Data structure to represent call participant identifiers
+   */
+  interface ICallParticipantIdentifiers {
+    /**
+     * Phone number of a PSTN caller
+     */
+    phoneNumber?: string;
```

Comment:
Since you made a custom class for EmailAddress (nice!), why did you choose not to make one for phone number? Seems like you could take a similar approach with a regex like:

`/^[\+]?[0-9]{0,3}\W?+[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}$/im`

Reply:
Thanks! I included it upon Aaron's suggestions

The reason I didn't add it for phone numbers is because our service doesn't pre-process phone numbers, we return the number in the format it comes in as. There's no fixed / expected format for phone numbers within our service, and we also don't want to coerce it into a specific format that might work for one customer or scenario but not for others.